x-pack/plugins/ingest_pipelines/public/application/components/pipeline_processors_editor/components/processor_form/processors/script.tsx

```diff
@@ -129,7 +129,7 @@ const fieldsConfig: FieldsConfig = {
 
 export const Script: FormFieldsComponent = ({ initialFieldValues }) => {
   const [showId, setShowId] = useState(() => !!initialFieldValues?.id);
-  const [scriptLanguage, setScriptLanguage] = useState<string>('plaintext');
+  const [scriptLanguage, setScriptLanguage] = useState<string>(PainlessLang.ID);
```

Comment:
The default script language in ES is "painless" so setting the `scriptLanguage` initial state to this as well.